de.json

```
@@ -353,7 +353,7 @@
 
         "Undo.SetField" : "Setze Feld {field_name} auf {value}",
         "Undo.SetReference" : "Setze Referenz {ref_name} auf {ref_target}",
-        "Undo.Spawn" : "Spawn {name}",
+        "Undo.Spawn" : "Erzeuge {name}",
```

```suggestion
        "Undo.Spawn" : "Erstelle {name}",
```
Probably better to use "Erstelle" here, like in other places (e.g. CreateNew). "Erzeuge" is more close to "generate".

---

I agree with @Avunia